content/writing/markdown.md

```diff
@@ -61,16 +62,22 @@ Do not leave any empty line between the header and the content after it.
 You’ll need a feature image: the image will be used on the front page, and as the hero image on your blog post. The size needs to be around 2:1 (twice larger than the height) to fit our WordPress template. The size of the images in the article doesn’t matter, but always use the highest possible quality: they will be automatically optimized using a lossless encoding.
 
 ### ALT Description & Caption
-Always add a text for the ALT description (see below) and a caption to all of the images.
+Always add a text for the ALT description and a caption to all of the images.
 
 `![this is where your alt text needs to be](myimage.png "this is where the caption goes")`
 
-Be as descriptive as you can: ask yourself how you would like someone to describe them to you if you were blind. Try to find a good balance between adding enough information vs having something way too long. Avoid adding words like pictures or photos. For example:
+Write in such a way that someone could totally understand without needing images. Consider the image as an addition for visual people, but not as a primary item that needs to be there, so people can understand what you want them to do or understand. In other words, you can remove the images completely and understand the tutorial or documentation. Doing this ensure that there is no difference in terms of information that visual capable people have vs the others. 
+
+#### Tips In Writing ALT Description
+1. Be as descriptive as you can but moderate: ask yourself how you would like someone to describe them to you if you were blind. Try to find a good balance between adding enough information vs having something way too long, since people using screen reader already have a lot of “reading” happening on the page, but add enough context so people understand. Avoid adding words like pictures or photos. For example:
```

Comment:
if you were blind = if you had impaired vision

since it's not just blind people who struggle to read on screen, it's more inclusive.